Syndicate Ninja

yackemflam opened the pull request:
:cl: Yackemflam
add: The syndicate has learned on how to give their agents a true syndicate ninja experience.
/:cl:

yackemflam committed:
Syndicate Ninja

ChangelingRain commented on code/game/objects/items/weapons/storage/uplink_kits.dm:
```diff
@@ -131,6 +131,19 @@
 			return
+		
+		if("ninja") // 33 tc worth
+			new /obj/item/weapon/katana(src) // Unique , hard to tell how much tc this is worth. 8 tc?
```
It's an esword you can't hide 5-6 tc

yackemflam replied:
It's better than an esword in terms of damage, 40 force

jughu replied:
Its also a emagh

KorPhaeron replied:
why the fuck did github change its format again

jughu replied:
Idk

ShadowDeath6 replied:
not the ekatana, it doesn't have emag powers

PKPenguin321 commented:
>this PR again

what have you changed since last time to make it not shit? or are you just pushing the same denied PR and hoping the guy that closed it last time wont notice?

yackemflam commented:
I added in the chameleon projector.
I also used the original katana instead of a weaker variant.
I also removed the mallninja memes since no one liked it.

Razharas commented:
Its fine

nullbear commented on code/game/objects/items/weapons/storage/uplink_kits.dm:
```diff
@@ -131,6 +131,19 @@
+			new /obj/item/weapon/throwing_star(src)
+			new /obj/item/weapon/throwing_star(src)
+			new /obj/item/weapon/throwing_star(src)
+			new /obj/item/weapon/throwing_star(src)
```
Someone is really bad at counting.

yackemflam replied:
I made that comment when there was 5 throwing stars and forgot to change it later, whooops

yackemflam committed:
Update uplink_kits.dm

KorPhaeron commented:
Getting a giant weapon that you can't hide seems sort of undesirable as traitor

PKPenguin321 reviewed (commented):
apparently i have to make a review if i want to comment on code at all
???
